ui/src/environments/environment-type.ts

```diff
@@ -37,6 +37,10 @@ export interface Environment {
   // The base URL for the Zendesk help center / user forum.
   // Example value: https://aousupporthelp.zendesk.com/hc/
   zendeskHelpCenterUrl: string;
+  // The Shibboleth URL for linking ERA Commons accounts
+  shibbolethUrl: string;
```

Comment:
Super nitpick: at least in this file, try to keep keys alphabetized so things are easy to find. (I've been though this before... alpha is the only way to sort things like this without it becoming a complete mess, as configs like this would otherwise do.)